gradle.properties

```diff
@@ -1,20 +1,23 @@
 # Done to increase the memory available to gradle.
-org.gradle.jvmargs=-Xmx1G
+org.gradle.jvmargs=-Xmx2048M
 org.gradle.parallel=true
 
 # Fabric Properties
 # check these on https://modmuss50.me/fabric.html
-minecraft_version=1.21
-yarn_mappings=1.21+build.7
-loader_version=0.15.11
+minecraft_version=1.21.3
+yarn_mappings=1.21.3+build.2
+loader_version=0.16.9
+
+# Fabric API
+fabric_version=0.108.0+1.21.3
```

Comment:
Please keep it with mod dependencies, as this is a mod dependency. No need to treat the Fabric API differently